src/zenml/integrations/aws/orchestrators/sagemaker_orchestrator.py

```diff
@@ -226,11 +246,20 @@ def get_pipeline_run_metadata(
         Returns:
             A dictionary of metadata.
         """
-        # TODO: Add this once we can get the region
-        # run_url = (
-        #     f"https://{region}.console.aws.amazon.com/sagemaker/"
-        #     f"home?region={region}"
-        # )
+        try:
+            region_name = self._get_region_name()
+        except RuntimeError:
+            region_name = ""
+
+        aws_run_id = os.environ[ENV_ZENML_SAGEMAKER_RUN_ID].split("/")[-1]
+        orchestrator_logs_url = (
+            f"https://{region_name}.console.aws.amazon.com/"
```

Comment:
in case `region_name` is "" then `orchestrator_logs_url` would be misformatted.. maybe return in the except above